### ISSUE TITLE
Issue with ImmediatelyFast

### ISSUE THREAD

**Issue by PerhapsAverage** [xCollateral/VulkanMod#260]
The UI is completely bugged out when using ImmediatelyFast. Skins and inventory, mainly, though skins dont break often. I'm not too sure about mobs. I tested it out with sodium and everything worked fine.
![image](https://github.com/xCollateral/VulkanMod/assets/137153984/2b0da40e-5197-40a4-972a-752d0b98395b)
![image](https://github.com/xCollateral/VulkanMod/assets/137153984/99b2ce56-4191-4a73-b1e0-cb7cd3df6cf4)
![image](https://github.com/xCollateral/VulkanMod/assets/137153984/f3b0c74e-f246-43a9-9e6f-336ea314ef11)
^^^^^ Image taken in F5 mode

**Comment by khnls** [comment]
Have you tried disabling hud_Batching in immediatelyFast's config file?

**Comment by khnls** [comment]
Try disabling hud_Batching in the ImmediatelyFast config file

**Comment by Zorklis** [comment]
Yeah it's incompatible, I think IF creator abandoned their support for Vulkan too

**Comment by andreasdc** [comment]
I think someone got IF to be compatible with VulkanMod.

**Comment by Versatile-V1** [comment]
@PerhapsAverage This can be closed as IF now mostly works with vulkanmod
